cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -763,9 +763,16 @@ private void logOperation() {
       // Populate original response of completed task if requested so.
       if (fetchCompletedTask) {
         try {
+          // TODO: For CompletedWithError tasks the cached response should contain the original server side error information
           jsonObjectMap.put(ORIGINAL_RESPONSE, lastFuture().get().cachedResponse());
         } catch (InterruptedException | ExecutionException e) {
-          throw new IllegalStateException("Error happened in fetching response for task " + _userTaskId.toString(), e);
+          if (_state == TaskState.COMPLETED_WITH_ERROR){
```

Comment:
> So checkstyle failed, but the line it is complaining about doesn't exit? How do I run the checkstyle task locally?

@tomncooper This is due to a missing space before `{` in this line:
```
> Task :cruise-control:checkstyleMain
[ant:checkstyle] [ERROR] /home/circleci/workspace/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java:769:56: '{' is not preceded with whitespace. [WhitespaceAround]
```

Reply:
Great, thanks.